addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComLighting4Message.java

```diff
@@ -74,43 +80,56 @@ public static SubType fromByte(int input) throws RFXComUnsupportedValueException
     }
 
     public enum Commands {
-        UNDEFINED_0(0),
-        ON(1),
-        UNDEFINED_2(2),
-        UNDEFINED_3(3),
-        OFF(4);
+        OFF_0(0, false),
+        ON_1(1, true),
+        OFF_2(2, false),
+        ON_3(3, true),
+        OFF_4(4, false),
+        ON_5(5, true),
+        ON_7(7, true),
+        ON_9(9, true),
+        ON_12(12, true),
+        UNKNOWN(-1, false);
```

Comment:
Yes, I understand. The problem is that I have to add 50 additional enum values (to cover the range 0-63) of which I don't know the meaning, for now they are only / especially useful when sending/receiving command-id's, because if we would throw the exception you could not even receive an commandId anymore.

https://community.openhab.org/t/help-with-rfxcom-and-4-port-relay-lighting4-protocol/9648